dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/configurations/ImportVariantsStepConfigurationTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbsnpimporter.configurations;
+
+import com.mongodb.BasicDBList;
+import com.mongodb.DBCollection;
+import com.mongodb.DBObject;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.batch.core.JobExecution;
+import org.springframework.batch.core.JobInstance;
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.explore.JobExplorer;
+import org.springframework.batch.test.JobLauncherTestUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.mongodb.core.MongoOperations;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import uk.ac.ebi.eva.dbsnpimporter.Parameters;
+import uk.ac.ebi.eva.dbsnpimporter.test.DbsnpTestDatasource;
+import uk.ac.ebi.eva.dbsnpimporter.test.configurations.JobTestConfiguration;
+import uk.ac.ebi.eva.dbsnpimporter.test.configurations.MongoTestConfiguration;
+
+import javax.sql.DataSource;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static uk.ac.ebi.eva.dbsnpimporter.configurations.ImportVariantsJobConfiguration.IMPORT_VARIANTS_JOB;
+
+@RunWith(SpringRunner.class)
+@TestPropertySource({"classpath:application.properties"})
```

Comment:
is this reading src/test/resources or src/main/resources properties file ?

Reply:
at least the test one for sure. I can conceive that spring is also reading the main one, but it is overwritten.